cupy/sparse/csr.py

```diff
@@ -60,6 +62,10 @@ def get(self, stream=None):
         return scipy.sparse.csr_matrix(
             (data, indices, indptr), shape=self._shape)
 
+    def _convert_dense(self, x):
+        m = cusparse.dense2csr(x)
```

Comment:
Another one is created to initialize a CSR matrix, a little entangled. An option is to make cusparse.py independent of cupy.sparse.spmatrix suites, but it is followed with some amount of code change. Too much?